### PR TITLE
changes needed for proper operation of self-hosting

### PR DESCRIPTION
**Pull request by jooni22**
## Please indicate what issue this PR is related to and @ any maintainers who are relevant
changes needed for proper operation of self-hosting in Hetzner cloud
@skeptrunedev

### CODE REVIEW

**Review by cdxker — CHANGES_REQUESTED** [review]
Looks great thanks for your contribution, we will need to also update the docker-compose.yml for the frontends to import the env's. I'll try to add it or you can if you have some time avail.

File: .env.example [hunk 1]
```diff
@@ -1,27 +1,23 @@
 ##### DOCKER-COMPOSE #####
-
 MINIO_ROOT_USER="rootuser"
 MINIO_ROOT_PASSWORD="rootpassword"
 REDIS_PASSWORD="thisredispasswordisverysecureandcomplex"
 HF_TOKEN="hf_"
```
Comment (cdxker) [inline .env.example:5]:
```suggestion
```
This can now be removed, no longer used

File: .env.example [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
 VITE_API_HOST="http://localhost:8090/api"
 VITE_SEARCH_UI_URL="http://localhost:5174"
 VITE_CHAT_UI_URL="http://localhost:5175"
 VITE_ANALYTICS_UI_URL="http://localhost:5176"
 VITE_SENTRY_DASHBOARD_DSN=""
-
-##### CHAT #####
+VITE_BM25_ACTIVE=true
 VITE_FRONTMATTER_VALS="link,tag_set,time_stamp,location"
 VITE_SENTRY_CHAT_DSN=""
```
Comment (cdxker) [inline .env.example:19]:
```suggestion
```
Same with this, just adds extra noise

File: .env.example [hunk 3]
```diff
@@ -1,27 +1,23 @@
...
 VITE_DASHBOARD_URL="http://localhost:5173"
-
-##### SEARCH #####
 VITE_SENTRY_SEARCH_DSN=""
```
Comment (cdxker) [inline .env.example:21]:
```suggestion
```
As well as this too.

File: .env.example [hunk 4]
```diff
@@ -1,27 +1,23 @@
 ##### DOCKER-COMPOSE #####
-
 MINIO_ROOT_USER="rootuser"
 MINIO_ROOT_PASSWORD="rootpassword"
 REDIS_PASSWORD="thisredispasswordisverysecureandcomplex"
 HF_TOKEN="hf_"
 ADMIN_API_KEY="admin"
 COMPOSE_PROFILES="debug"
```
Comment (cdxker) [inline .env.example:7]:
```suggestion
```
This also adds extra noise

**Review by cdxker — CHANGES_REQUESTED** [review]
I'm not able to commit on your branch, but I also requested a few simple changes that tidy up the .env.example file a little bit. 

As for the docker-compose.yml. I needed to add this diff to the docker-compose.yml
image

VITE_BM25_ACTIVE doesn't error if it doesn't exist, but it adds BM25 support to the UI which self-hosters won't know exists if its not visibile in the UI

**Review by jooni22 — COMMENTED** [review]
looks good

File: .env.example (hunk 3 above)
Comment (jooni22) [reply .env.example]: I thought of deleting and wrote of adding

**Review by cdxker — APPROVED** [review]
Sweet, merging!

**Pull request merged**